keras/src/backend/openvino/numpy.py

```diff
@@ -836,7 +836,15 @@ def log(x):
 
 
 def log10(x):
-    raise NotImplementedError("`log10` is not supported with openvino backend")
+    x = get_ov_output(x)
+    x_type = x.get_element_type()
+    if x_type.is_integral():
+        ov_type = OPENVINO_DTYPES[config.floatx()]
+        x = ov_opset.convert(x, ov_type)
+    log_x = ov_opset.log(x).output(0)
+    log_10 = ov_opset.constant(np.log(10), log_x.get_element_type()).output(0)
```

Comment:
please better to create constant 10 of type `x_type` and compute ov log instead using numpy